source/azure-pipelines/ado_agent_networking.html.md.erb

```diff
@@ -0,0 +1,96 @@
+---
+title: Agent Pool Networking
+weight: 30
+---
+
+# <%= current_page.data.title %>
+ADO pipelines, agent pools are containers hosted by Azure Kubernetes services for HMCTS. 
+
+Agent Capabilities can help narrow down the vNet hosting the container. First the AZP_Name follows a manual naming convention. Try removing the HMCTS at the begining and adding VNET to the end.
+If the agent is actually a Kubernetes container, it will also state the Kubernetes_Port, with HMCTS being 10.0.0.1:443.
+
+![Agent_Capabilities](images/ADO-Agent-Capabilities.png )
+
+### Virtual Networks
+These are examples of some of the agent pool and vNet names
+
+| Agent Pool | Virtual network | subnet| IPv4 |
+| - | - | - | - |
+| hmcts-ss-ptl | ss-ptl-vnet | aks-00 | 10.147.64.0/20 |
```

Comment:
Not sure if we want to store subnet information as it could change and become out of date